app/models/requests/AuthorisationRequest.scala

```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  */
 
-package models
+package models.requests
```

Comment:
This change should not be done as part of this Change Request because it touches too many files obfuscating the PR

Reply:
resolved